docs/keycodes.md

```diff
@@ -256,6 +256,8 @@ This is a reference only. Each group of keys links to the page documenting their
 |----------------------------------|---------|------------------------------------|
 |`MAGIC_SWAP_CONTROL_CAPSLOCK`     |         |Swap Caps Lock and Left Control     |
 |`MAGIC_CAPSLOCK_TO_CONTROL`       |         |Treat Caps Lock as Control          |
+|`MAGIC_SWAP_LCTL_LGUI`            |         |Swap Left Control and GUI           |
```

Comment:
Please also add these to https://docs.qmk.fm/#/feature_bootmagic :)

Reply:
This still needs to be done.  

And it's /docs/feature_bootmagic.md in this repo.